flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/nfa/SharedBuffer.java

```diff
@@ -519,8 +498,9 @@ public int hashCode() {
 		private final ValueTimeWrapper<V> valueTime;
 		private final Set<SharedBufferEdge<K, V>> edges;
 		private final SharedBufferPage<K, V> page;
+
 		private int referenceCounter;
-		private transient int entryId;
+		private int entryId;
```

Comment:
Why is this not `transient` anymore?

Reply:
we were not serializing it anyway and the class is not `Serializable`.